desktopexporter/internal/telemetry/logs.go

```diff
@@ -11,18 +14,27 @@ type LogsPayload struct {
 }
 
 type LogData struct {
-	Body                   string                 `json:"body,omitempty"`
-	TraceID                string                 `json:"traceID,omitempty"`
-	SpanID                 string                 `json:"spanID,omitempty"`
-	Timestamp              time.Time              `json:"timestamp,omitempty"`
-	ObservedTimestamp      time.Time              `json:"observedTimestamp,omitempty"`
-	Attributes             map[string]interface{} `json:"attributes,omitempty"`
-	SeverityText           string                 `json:"severityText,omitempty"`
-	SeverityNumber         plog.SeverityNumber    `json:"severityNumber,omitempty"`
-	DroppedAttributesCount uint32                 `json:"droppedAttributeCount,omitempty"`
-	Flags                  plog.LogRecordFlags    `json:"flags,omitempty"`
-	Resource               *ResourceData          `json:"resource"`
-	Scope                  *ScopeData             `json:"scope"`
+	Timestamp			   int64 `json:"-"`
+	ObservedTimestamp	   int64 `json:"-"`
+
+	TraceID                string `json:"traceID,omitempty"`
+	SpanID                 string `json:"spanID,omitempty"`
+	
+	SeverityText           string `json:"severityText,omitempty"`
+	SeverityNumber         int32  `json:"severityNumber,omitempty"`
+	
+	Body                   any            `json:"body,omitempty"`
+	Resource               *ResourceData  `json:"resource"`
+	Scope                  *ScopeData     `json:"scope"`
+	Attributes             map[string]any `json:"attributes,omitempty"`
+	DroppedAttributesCount uint32         `json:"droppedAttributeCount,omitempty"`
+	Flags             	   uint32         `json:"flags,omitempty"`
+	EventName              string         `json:"eventName,omitempty"`
```

Comment:
Some weird spacing. This could need a `gofmt`

Reply:
Note mostly to self because I will forget and break my editor settings again in the future: For gofmt to work on save, we need `"editor.formatOnSaveMode": "file"`